buildpacks/gradle/src/detect.rs

```diff
@@ -0,0 +1,17 @@
+use std::path::Path;
+
+pub(crate) fn is_gradle_project_directory(root_dir: &Path) -> bool {
+    // We look for these Gradle specific files and not 'gradlew' directly. This allows us to
+    // fail with an error message explaining that 'gradlew' is required for Gradle projects.
+    // If we just fail detect on a missing Gradle Wrapper, we lose the opportunity to display
+    // such a message, worsening DX.
+    [
+        "build.gradle",
+        "settings.gradle",
+        "build.gradle.kts",
+        "settings.gradle.kts",
+    ]
+    .into_iter()
+    .map(|file_name| root_dir.join(file_name))
+    .any(|path| path.exists())
```

Comment:
Should the check here be using `try_exists()` instead, so IO errors result in a visible failure rather than silently not running the buildpack? (xref https://github.com/heroku/buildpacks-python/blob/1ea5938dcfe66fc2e4e048265201558999a72fb0/src/utils.rs#L28-L37)

Reply:
I don't think is is a huge issue, but it's definately better to use `try_exists`. 86acd1f changes this to `try_exists`.